php/xdebug.ini

```diff
@@ -0,0 +1,4 @@
+[XDebug]
```

Comment:
Please in the first line add #ddev-generated, so this file can be updated by aljibe in the future.